aind-labtracks-service-server/README.md

```diff
@@ -7,5 +7,73 @@
 ![Coverage](https://img.shields.io/badge/coverage-100%25-brightgreen?logo=codecov)
 ![Python](https://img.shields.io/badge/python->=3.10-blue?logo=python)
 
-REST service to retrieve data from an backend.
+REST service to retrieve data LabTracks.
 
+## Local Development
+
+Requires docker to build and run package locally.
+
+- Create a file called env/webapp.env with appropriate env variables.
```

Comment:
it'd be useful to have an env.template (ex: "LABTRACKS_SERVER" vs "LABTRACKS_HOST")